README.md

```diff
@@ -62,21 +45,26 @@ The code has been compiled and tested with
 clang version 7.0 and 8.0
 using either GNU's libstdc++ or LLVM's libc++.
 
-### LLVM dependency
 
-F18 uses components from LLVM.
+## Building Flang out of tree
+These instructions are for building Flang separately from LLVM; if you are
+building Flang alongside LLVM then follow the standard LLVM build instructions
```

Comment:
I think you need this statement in the section above about Building Flang in-tree too.

Reply:
I think in general if you're building in-tree you're not going to read this README and will just follow the normal build instructions? I can change this though, where would you put it?